chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomTest.kt

```diff
@@ -0,0 +1,65 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.JoinedRoom
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class JoinedRoomTest : Spek({
+
+    describe("given no rooms") {
+        val givenState = ChatkitState(joinedRoomsState = JoinedRoomsState(
+            rooms = emptyMap(),
+            unreadCounts = emptyMap()
+        ))
+
+        describe("when a room is joined") {
+            val joinedRoom = JoinedRoom(roomOne, unreadCount = 1)
+            val newState = joinedRoomReducer(givenState, joinedRoom)
+
+            it("then the state should contain the expected room") {
```

Comment:
```suggestion
            it("then the state contains the expected room") {
```
how about removing "should"s as suggested [here](https://github.com/pusher/chatkit-android/pull/192#discussion_r387154113)?

Reply:
👍 have removed all the shoulds from the tests